src/cli/main.c

```diff
@@ -280,6 +314,41 @@ static errno_t show(struct cli_cmdline *cmdline)
     return EOK;
 }
 
+static errno_t requirements(struct cli_cmdline *cmdline)
+{
+    struct authselect_profile *profile;
+    const char *profile_id;
+    const char **features;
+    const char *requirements;
+    errno_t ret;
+
+    ret = parse_profile_options(cmdline, NULL, &profile_id, &features);
+    if (ret != EOK) {
+        return ret;
+    }
+
+    ret = authselect_profile(profile_id, &profile);
+    if (ret != EOK) {
+        ERROR("Unable to get profile information [%d]: %s",
+              ret, strerror(ret));
+        return ENOMEM;
+    }
+
+    requirements = authselect_profile_requirements(profile, features);
```

Comment:
Also here it looks like requirements are not freed

Reply:
Same as  above.